### PR TITLE
21.1.5 update

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# WPF Dock Layout Manager - Limit LayoutGroup Customization at Runtime
+
+This example limits [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) customization at runtime.
+
+<img src="https://user-images.githubusercontent.com/12169834/175359371-47c8a0aa-9d45-470d-9f9f-a830ac39a423.png" width=900px/>
+
+This example contains a [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) class descendant and the attached `DisableCrossingGroupBoundaries` property. You can attach the `DisableCrossingGroupBoundaries` property only to the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup).
```
Comment (albertov05) [inline Readme.md]:
```suggestion
The example contains a [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) class descendant and the attached `DisableCrossingGroupBoundaries` property. You can attach the `DisableCrossingGroupBoundaries` property only to the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup).
```

File: Readme.md [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+
+In this case, the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) with the property assigned to `true` allows any customization and prevents moving inner elements outside [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) boundaries. For the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) without this property, the default logic is used for arranging elements.
```
Comment (albertov05) [inline Readme.md]: 1. `In this case, the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) with the property assigned to `true` allows any customization and prevents moving inner elements outside [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) boundaries.`

a) What do you mean by "any customization"? Is it customization by users or the developer?
b) Who is prevented from moving inner elements outside, a user or a developer?

2. `For the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) without this property, the default logic is used for arranging elements.`

Please change "the LayoutGroup" to "a LayoutGroup".

Comment (friedcucumber) [reply Readme.md]:
```suggestion
In this case, a [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) with the property assigned to `true` allows users to customize the group and prevents them moving inner elements outside [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) boundaries. For the [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) without this property, the default logic is used for arranging elements.

```

Comment (albertov05) [reply Readme.md]: I don't think "prevents them moving" is correct. Can you change it to "prevents them from moving"?

**Pull request merged**